packages/fhir-converter/src/templates/cda/DataType/CodeableConcept.hbs

```diff
@@ -1,6 +1,8 @@
 {
 	{{#if code.originalText}}
-		"text":"{{{code.originalText._}}}",
+		"text":"{{{parseReferenceData code.originalText._ }}}",
+	{{else if text}}
+		"text": "{{{parseReferenceData text}}}",
```

Comment:
added optional argument here of text variable to be passed in so that Observation can use CodeableConcept instead of rewriting the same primitive again